notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb

```diff
@@ -0,0 +1,2444 @@
+{
```

Comment:
<p>Change from master branch to main branch</p><p> </p><p>Add H1 title above the links</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>